packages/types/src/parser-options.ts

```diff
@@ -17,7 +17,8 @@ type EcmaVersion =
   | 2017
   | 2018
   | 2019
-  | 2020;
+  | 2020
+  | 'latest';
```

Comment:
while we're here - might as well add these two in as well

```ts
  | 12
  // ...
  | 2021
  | 'latest';
```

Reply:
Added `2021` and `2022`

Reply:
let's remove `latest` from here and add it to the `parser` types as we're now handling it specifically within the parser